### PR TITLE
Create square_root_using_netwon_repson.py

### PR DESCRIPTION
**Pull request by Lablnet**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/square_root_using_netwon_repson.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Finding Square Roots Using Newton's(Newton–Raphson) Method
+n = input("Enter a number to find square root: ")
```
Comment (cclauss) [inline maths/square_root_using_netwon_repson.py]: This line causes our automated testing to fail.  See CONTRIBUTING.md.

Please encapsulate your algorithm into a function that could be dropped into a larger program:
`def sqrt_using_newton_raphson(n: float) -> float:`

Please add a few doctests that compare the results with `math.sqrt()`.  Your function should behave just like `math.sqrt()` including using the same exceptions and exceptions messages.

**Comment by onlinejudge95** [comment]
Also, the file name should be `square_root_using_newton_raphson.py`

**Comment by TheSuperNoob** [comment]
It also looks like [maths/square_root.py](https://github.com/TheAlgorithms/Python/blob/master/maths/square_root.py) is using the exact same algorithm. Not sure what the policy for duplicate algorithms is here.

**Comment by cclauss** [comment]
If this algorithm is exactly the same as an existing algorithm then this PR should be closed especially given https://travis-ci.com/TheAlgorithms/Python/builds/149131996#L820

**Pull request closed without merging**